### DISCUSSION TITLE
[Explorations] Evaluate content and display

### DISCUSSION THREAD

**Discussion by gusaus** [OpenProducer/angelcityjazz#64] [Planning & Strategy]
Currently, we have this landing page containing the following types of posts -
http://box2342.temp.domains/~angeldd8/explorations/

- LA Founders -
  - https://content-angelcityjazz.pantheonsite.io/explorations/ornette-coleman/
  - https://content-angelcityjazz.pantheonsite.io/explorations/billy-higgins/
- Watch/Video - see https://github.com/OpenProducer/angelcityjazz/issues/9
- Listen - currently, the Spotify embeds in the sidebar

 Couple thoughts when looking at this now - 
- This may work better as one column wide with the Spotify embeds in a third column
- what if each Spotify playlist was embedded in its own post; somewhat similar to the archival video posts
- I like what you're trying to do with the LA Founders posts - I'll do some touch-up based on your comments https://github.com/OpenProducer/angelcityjazz/issues/10#issuecomment-674194937 after I'm done with migration
- As with videos https://github.com/OpenProducer/angelcityjazz/issues/9#issuecomment-671665593 we need to come up with consistent categories and tags 

Any other thoughts, feedback, and suggestions?

**Comment by gusaus** [comment]
Couple notes on the founders pages  https://github.com/OpenProducer/angelcityjazz/issues/10#issuecomment-674194937

1 - the html block you used to embed spotify doesn't seem to keep after a page is edited - for example -
https://content-angelcityjazz.pantheonsite.io/explorations/billy-higgins/

I'll troubleshoot that

2 - think it might be better to provide text links (rather than social icons) for artists - I could format a block similar to https://livesessions.npr.org/artists/kamasi-washington

Will try a few things out on https://theme-angelcityjazz.pantheonsite.io/ when back online later this afternoon.

**Comment by gusaus** [comment]
So the Spotify embed does work for playlists - check out the configuration settings - 
https://angelcityjazz.com/wp-admin/post.php?post=1687&action=edit

![Edit Post ‹ Angel City Jazz — WordPress 2020-08-14 19-06-35](https://user-images.githubusercontent.com/113996/90303274-8d2dbb80-de61-11ea-821c-aba33f408234.png)

**Comment by roccosomazzi** [comment]
I removed the playlist form the event pages. Seemed out of place there. It
would be great to integrate it in the festival main page instead

On Fri, Aug 14, 2020 at 7:09 PM Gus Austin <notifications@github.com> wrote:

> So the Spotify embed does work for playlists - check out the configuration
> settings -
>
> https://content-angelcityjazz.pantheonsite.io/wp-admin/post.php?post=1687&action=edit
>
> [image: Edit Post ‹ Angel City Jazz — WordPress 2020-08-14 19-06-35]
> <https://user-images.githubusercontent.com/113996/90303274-8d2dbb80-de61-11ea-821c-aba33f408234.png>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-674333656>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3CYAW6H64WEKZOF55LSAXU6BANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by roccosomazzi** [comment]
Also, not sure what you changed on the exploration pages but the playlist
now is too narrow. I think it looked better as I had it before. You
should be able to read the full title.

On Fri, Aug 14, 2020 at 10:12 PM Rocco Somazzi <roccosomazzi@gmail.com>
wrote:

> I removed the playlist form the event pages. Seemed out of place there. It
> would be great to integrate it in the festival main page instead
>
> On Fri, Aug 14, 2020 at 7:09 PM Gus Austin <notifications@github.com>
> wrote:
>
>> So the Spotify embed does work for playlists - check out the
>> configuration settings -
>>
>> https://content-angelcityjazz.pantheonsite.io/wp-admin/post.php?post=1687&action=edit
>>
>> [image: Edit Post ‹ Angel City Jazz — WordPress 2020-08-14 19-06-35]
>> <https://user-images.githubusercontent.com/113996/90303274-8d2dbb80-de61-11ea-821c-aba33f408234.png>
>>
>> —
>> You are receiving this because you were assigned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-674333656>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQGKU3CYAW6H64WEKZOF55LSAXU6BANCNFSM4P7Y3N2A>
>> .
>>
>
>
> --
> Rocco Somazzi
>
> http://angelcityarts.org
> http://angelcityjazz.com
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by roccosomazzi** [comment]
Should I keep editing on the content site or wait until it's on bluehost?

On Fri, Aug 14, 2020 at 10:14 PM Rocco Somazzi <roccosomazzi@gmail.com>
wrote:

> Also, not sure what you changed on the exploration pages but the playlist
> now is too narrow. I think it looked better as I had it before. You
> should be able to read the full title.
>
> On Fri, Aug 14, 2020 at 10:12 PM Rocco Somazzi <roccosomazzi@gmail.com>
> wrote:
>
>> I removed the playlist form the event pages. Seemed out of place there.
>> It would be great to integrate it in the festival main page instead
>>
>> On Fri, Aug 14, 2020 at 7:09 PM Gus Austin <notifications@github.com>
>> wrote:
>>
>>> So the Spotify embed does work for playlists - check out the
>>> configuration settings -
>>>
>>> https://content-angelcityjazz.pantheonsite.io/wp-admin/post.php?post=1687&action=edit
>>>
>>> [image: Edit Post ‹ Angel City Jazz — WordPress 2020-08-14 19-06-35]
>>> <https://user-images.githubusercontent.com/113996/90303274-8d2dbb80-de61-11ea-821c-aba33f408234.png>
>>>
>>> —
>>> You are receiving this because you were assigned.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-674333656>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AQGKU3CYAW6H64WEKZOF55LSAXU6BANCNFSM4P7Y3N2A>
>>> .
>>>
>>
>>
>> --
>> Rocco Somazzi
>>
>> http://angelcityarts.org
>> http://angelcityjazz.com
>>
>
>
> --
> Rocco Somazzi
>
> http://angelcityarts.org
> http://angelcityjazz.com
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
> not sure what you changed on the exploration pages but the playlist now is too narrow. I think it looked better as I had it before. You should be able to read the full title.

We currently using the same two-column template as events and pages https://github.com/OpenProducer/angelcityjazz/issues/10

I think it would work better using the same one column wide template as the other landing pages w/ an additional column... or putting in a container underneath.

Here's an initial mockup of the former https://theme-angelcityjazz.pantheonsite.io/explorations/

**Comment by roccosomazzi** [comment]
The only thing I don’t like now is that the spotify playlist is to narrow
and there is a lot of ampty space around it.

On Sat, Aug 15, 2020 at 2:01 PM Gus Austin <notifications@github.com> wrote:

>
>
>
>
> not sure what you changed on the exploration pages but the playlist now is
> too narrow. I think it looked better as I had it before. You should be able
> to read the full title.
>
>
>
>
> We currently using the same two-column template as events and pages #10
> <https://github.com/OpenProducer/angelcityjazz/issues/10>
>
>
> I think it would work better using the same one column wide template as
> the other landing pages w/ an additional column... or putting in a
> container underneath.
>
>
> Here's an initial mockup of the former
> https://theme-angelcityjazz.pantheonsite.io/explorations/
>
>
>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-674446776>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3H25IJXH5RUZI7QB7LSA3ZQ3ANCNFSM4P7Y3N2A>
> .
>
>
> --
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
I can fix that w/ CSS

**Comment by roccosomazzi** [comment]
I also changed the links to text per your suggestion

On Sat, Aug 15, 2020 at 2:11 PM Gus Austin <notifications@github.com> wrote:

>
>
> I can fix that w/ CSS
>
>
>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-674447589>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3CGBNL4QGS6YFW5DD3SA32WZANCNFSM4P7Y3N2A>
> .
>
>
> --
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
Might work better in underneath the spotify embed (which also needs to be resized w/CSS). 

I copied these over to the production site where we can continue to modify - 
http://box2342.temp.domains/~angeldd8/explorations/billy-higgins/
http://box2342.temp.domains/~angeldd8/explorations/ornette-coleman/

You'll probably notice a couple things that are different.

**Comment by roccosomazzi** [comment]
In the related section I only want other LA founders articles (there's
gonna be 5 or 6 of them).

On Sat, Aug 15, 2020 at 2:39 PM Gus Austin <notifications@github.com> wrote:

> Might work better in underneath the spotify embed (which also needs to be
> resized w/CSS).
>
> I copied these over to the production site where we can continue to modify
> -
> http://box2342.temp.domains/~angeldd8/explorations/billy-higgins/
> http://box2342.temp.domains/~angeldd8/explorations/ornette-coleman/
>
> You'll probably notice a couple things that are different.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-674449791>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3B4FKPFWCQVSOOWMU3SA3575ANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
Couple tweaks and suggestions based on comments in this post - 

>In the related section I only want other LA founders articles (there's gonna be 5 or 6 of them).

I disabled the Jetpack related content (again) as it doesn't give us granular control. Not sure we need the related content on LA Founders until there are more than two posts.

> The only thing I don’t like now is that the spotify playlist is to narrow and there is a lot of ampty space around it.

The Spotify embeds should now display the full width of the columns or containers. What do you think about having the links underneath the Spotify embed - 
https://theme-angelcityjazz.pantheonsite.io/explorations/billy-higgins/
https://theme-angelcityjazz.pantheonsite.io/explorations/ornette-coleman/

Also, here are a couple different looks on the explorations landing page - 

https://theme-angelcityjazz.pantheonsite.io/explorations/
https://theme-angelcityjazz.pantheonsite.io/explorations-2/

If any of those seem good, I can refine and then copy over to the live site.

Are there any changes we need on this landing page and associated content?

**Comment by roccosomazzi** [comment]
The LA founders pages look good, but please leave the other LA founders
pages at the bottom. I have at least 3 more I'm gonna add very soon.
Let's use https://theme-angelcityjazz.pantheonsite.io/explorations-2/
The only change I would do: since we're gonna add much more content on this
page can we display 4 articles and 4 archival video and have a 'more'
button to see the rest?

On Mon, Aug 17, 2020 at 3:23 PM Gus Austin <notifications@github.com> wrote:

> Couple tweaks and suggestions based on comments in this post -
>
> In the related section I only want other LA founders articles (there's
> gonna be 5 or 6 of them).
>
> I disabled the Jetpack related content (again) as it doesn't give us
> granular control. Not sure we need the related content on LA Founders until
> there are more than two posts.
>
> The only thing I don’t like now is that the spotify playlist is to narrow
> and there is a lot of ampty space around it.
>
> The Spotify embeds should now display the full width of the columns or
> containers. What do you think about having the links underneath the Spotify
> embed -
> https://theme-angelcityjazz.pantheonsite.io/explorations/billy-higgins/
> https://theme-angelcityjazz.pantheonsite.io/explorations/ornette-coleman/
>
> Also, here are a couple different looks on the explorations landing page -
>
> https://theme-angelcityjazz.pantheonsite.io/explorations/
> https://theme-angelcityjazz.pantheonsite.io/explorations-2/
>
> If any of those seem good, I can refine and then copy over to the live
> site.
>
> Are there any changes we need on this landing page and associated content?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-675144694>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3EZYZBSKDIXTAAMGTTSBGUU7ANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
I updated http://box2342.temp.domains/~angeldd8/explorations/ but haven't added backgrounds or anything. Might work well with just the white backgrounds? Also might be good to have a cover image, post, or slideshow spanning top column. Similar to what I was suggesting for the home page https://github.com/OpenProducer/angelcityjazz/issues/4#issuecomment-663298761

I think we can reformat when there's additional content. There are many displays we can generate with the Homepage blocks https://wordpress.com/support/wordpress-editor/blocks/blog-posts-block/ 

Also...if you noticed my comment about a video/watch landing page https://github.com/OpenProducer/angelcityjazz/issues/9#issuecomment-673105654, we could leave off or just display a few video archives once we have more content.

**Comment by roccosomazzi** [comment]
White background is fine. Not sure if a slider on top would work. I can of
like the simple layout as it is now. SHould I be editing on this site now?
http://box2342.temp.domains/~angeldd8/wp-admin/
I noticed that the links in these pages (Like
http://box2342.temp.domains/~angeldd8/wp-admin/post.php?post=1752&action=edit)
are still going to the old site (like
https://content-angelcityjazz.pantheonsite.io/explorations/ornette-coleman)
do we have to change all the links manually?

On Tue, Aug 18, 2020 at 1:39 AM Gus Austin <notifications@github.com> wrote:

> I updated http://box2342.temp.domains/~angeldd8/explorations/ but haven't
> added backgrounds or anything. Might work well with just the white
> backgrounds? Also might be good to have a cover image, post, or slideshow
> spanning top column. Similar to what I was suggesting for the home page #4
> (comment)
> <https://github.com/OpenProducer/angelcityjazz/issues/4#issuecomment-663298761>
>
> I think we can reformat when there's additional content. There are many
> displays we can generate with the Homepage blocks
> https://wordpress.com/support/wordpress-editor/blocks/blog-posts-block/
>
> Also...if you noticed my comment about a video/watch landing page #9
> (comment)
> <https://github.com/OpenProducer/angelcityjazz/issues/9#issuecomment-673105654>,
> we could leave off or just display a few video archives once we have more
> content.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-675344301>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3A2JDLNNQJP46FRCITSBI42LANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
Yes - we should be editing and testing http://box2342.temp.domains/~angeldd8/ 

Without a cover, I think http://box2342.temp.domains/~angeldd8/explorations/ is cleaner w/ the 'Explorations' title hidden (which is how it's currently set).

Links to other pages in posts should be checked and changed if needed.

There shouldn't be that many, no?

**Comment by gusaus** [comment]
Ok - the video and spotify embeds have been bothering me on the LA Creators posts - think I made some progress on the staging site (yes - Bluehost provides one) - The related post(s) still look out of place (until there are a few more), but I think the overall layout is a bit cleaner. If you generally agree (let me know what other improvements should be made) I can copy these posts over to https://angelcityjazz.com/

https://angelcityjazz.com/staging/9633/explorations/billy-higgins/
https://angelcityjazz.com/staging/9633/explorations/ornette-coleman/

**Comment by roccosomazzi** [comment]
Looks fine but maybe we can have a bit more margin in between the header
image and the text below. Also I do not want the social media links on the
header image. I will add a few more creator pages very soon (already have
the text)

On Wed, Aug 19, 2020 at 5:52 PM Gus Austin <notifications@github.com> wrote:

> Ok - the video and spotify embeds have been bothering me on the LA
> Creators posts - think I made some progress on the staging site (yes -
> Bluehost provides one) - The related post(s) still look out of place (until
> there are a few more), but I think the overall layout is a bit cleaner. If
> you generally agree (let me know what other improvements should be made) I
> can copy these posts over to https://angelcityjazz.com/
>
> https://angelcityjazz.com/staging/9633/explorations/billy-higgins/
> https://angelcityjazz.com/staging/9633/explorations/ornette-coleman/
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-676833838>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3GG6ZECVKGWZHEINSTSBRXVDANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by roccosomazzi** [comment]
Bty, I installed a new logo display plugin. Lots of options on how to
display them. Check out the logos at the bottom: https://angelcityjazz.com/

On Wed, Aug 19, 2020 at 6:06 PM Rocco Somazzi <roccosomazzi@gmail.com>
wrote:

> Looks fine but maybe we can have a bit more margin in between the header
> image and the text below. Also I do not want the social media links on the
> header image. I will add a few more creator pages very soon (already have
> the text)
>
> On Wed, Aug 19, 2020 at 5:52 PM Gus Austin <notifications@github.com>
> wrote:
>
>> Ok - the video and spotify embeds have been bothering me on the LA
>> Creators posts - think I made some progress on the staging site (yes -
>> Bluehost provides one) - The related post(s) still look out of place (until
>> there are a few more), but I think the overall layout is a bit cleaner. If
>> you generally agree (let me know what other improvements should be made) I
>> can copy these posts over to https://angelcityjazz.com/
>>
>> https://angelcityjazz.com/staging/9633/explorations/billy-higgins/
>> https://angelcityjazz.com/staging/9633/explorations/ornette-coleman/
>>
>> —
>> You are receiving this because you were assigned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-676833838>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQGKU3GG6ZECVKGWZHEINSTSBRXVDANCNFSM4P7Y3N2A>
>> .
>>
>
>
> --
> Rocco Somazzi
>
> http://angelcityarts.org
> http://angelcityjazz.com
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
Fine-tuned a bit and copied over -
https://angelcityjazz.com/explorations/billy-higgins/
https://angelcityjazz.com/explorations/ornette-coleman/

best way to keep the formatting consistent again is to clone and then change -  
<img width="1290" alt="LA Creators: Billy Higgins - Angel City Jazz 2020-08-19 19-34-08" src="https://user-images.githubusercontent.com/113996/90710418-335b3600-e253-11ea-9d91-40e263d62136.png">

**Comment by roccosomazzi** [comment]
Just noticed that the author is not showing. I want to make sure to credit
the writer like he was credited on the content site, both on the posts and
on the exploration page:
https://content-angelcityjazz.pantheonsite.io/explorations/billy-higgins/
https://content-angelcityjazz.pantheonsite.io/explorations/



On Wed, Aug 19, 2020 at 7:36 PM Gus Austin <notifications@github.com> wrote:

> Fine-tuned a bit and copied over -
> https://angelcityjazz.com/explorations/billy-higgins/
> https://angelcityjazz.com/explorations/ornette-coleman/
>
> best way to keep the formatting consistent again is to clone and then
> change -
> [image: LA Creators: Billy Higgins - Angel City Jazz 2020-08-19 19-34-08]
> <https://user-images.githubusercontent.com/113996/90710418-335b3600-e253-11ea-9d91-40e263d62136.png>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-676864997>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3D5KUPPN3H2YMLIP23SBSD4HANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityarts.org
http://angelcityjazz.com

**Comment by gusaus** [comment]
I enabled via the Customizer - 

<img width="1131" alt="Customize: LA Creators: Ornette Coleman - Angel City Jazz 2020-08-19 19-55-07" src="https://user-images.githubusercontent.com/113996/90711676-3277d380-e256-11ea-9887-da6f03288629.png">

**Comment by gusaus** [comment]
I noticed there's a new LA creators profile (looks great!) - I changed the grid display from 3 to 2 column - evens out the display of all for and they are less squished together on smaller screens and devices - https://angelcityjazz.com/explorations/

**Comment by roccosomazzi** [comment]
That looks good. On a related note, at the bottom of each creator page, in
the related posts grid, Ornette's page seems to be missing. I tried to
change the configuration but couldn't figure it out.


On Mon, Oct 12, 2020 at 11:33 AM Gus Austin <notifications@github.com>
wrote:

> I noticed there's a new LA creators profile (looks great!) - I changed the
> grid display from 3 to 2 column - evens out the display of all for and they
> are less squished together on smaller screens and devices -
> https://angelcityjazz.com/explorations/
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-707280922>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3BOY5HPRZLSNZG242LSKNDWBANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityjazz.com

**Comment by gusaus** [comment]
It's a reusable block - I changed the number of posts to 4; grid only shows 3 since it automatically hides the current post; also set the width to wide so it's consistent with other sections on the page - 

<img width="1329" alt="Edit Reusable Block ‹ Angel City Jazz — WordPress 2020-10-12 12-24-58" src="https://user-images.githubusercontent.com/113996/95785171-1ad32f00-0c8a-11eb-954e-91bc2cff87c9.png">

**Comment by roccosomazzi** [comment]
Eventually, we'll have many more. Any way to have a more button there?

On Mon, Oct 12, 2020 at 12:55 PM Gus Austin <notifications@github.com>
wrote:

> It's a reusable block - I changed the number of posts to 4; grid only
> shows 3 since it automatically hides the current post; also set the width
> to wide so it's consistent with other sections on the page -
>
> [image: Edit Reusable Block ‹ Angel City Jazz — WordPress 2020-10-12
> 12-24-58]
> <https://user-images.githubusercontent.com/113996/95785171-1ad32f00-0c8a-11eb-954e-91bc2cff87c9.png>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenProducer/angelcityjazz/issues/12#issuecomment-707315701>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGKU3A3XXOIKIAZH4VDCPTSKNNLJANCNFSM4P7Y3N2A>
> .
>


-- 
Rocco Somazzi

http://angelcityjazz.com

**Comment by gusaus** [comment]
We can enable in the reusable block settings - think we could enable when there's more than 3.

**Comment by gusaus** [comment]
Reopening for the 2021 festival - https://live-angelcityjazz.pantheonsite.io/explorations/

**Couple things based on feedback** 

- Per https://github.com/OpenProducer/angelcityjazz/issues/5#issuecomment-886835474 I added the 2021 Spotify playlist
- Per an email comment I removed the archival videos from the exploration page 

@angelcityjazz How does that look now?

**Comment by gusaus** [comment]
Following up on https://github.com/OpenProducer/angelcityjazz/issues/5#issuecomment-890363443

> I think all the post previews should be the same size. 

I think the newest article in the top section works - we could still tweak how it's displayed. 
https://angelcityjazz.com/staging/5481/explorations/

> THis goes for the playlists as well.

I put all the playlists in 2 columns but we'd need one more to balance out.

**Comment by angelcityjazz** [comment]
I think the new post should also be in a two column format. We will add
more posts so they should all be organized the same way. I don't mind the
fact that the playlists are uneven.

**Comment by gusaus** [comment]
How 'bout just this (with or without the header)?
https://angelcityjazz.com/staging/5481/explorations/

**Comment by gusaus** [comment]
...or what about keeping the layout similar to how it is now?
https://angelcityjazz.com/explorations2/

**Comment by angelcityjazz** [comment]
I think this works: https://angelcityjazz.com/staging/5481/explorations/
but we need to remove the LA Creators Profiles title. Not sure if we need a
new title or if we can just leave it off.

**Comment by gusaus** [comment]
I'd say leave it out
https://angelcityjazz.com/staging/5481/explorations/

or add back the generic 'Explorations' title - 
https://angelcityjazz.com/staging/5481/explorations2-2/

**Comment by angelcityjazz** [comment]
Either way works for me

**Comment by angelcityjazz** [comment]
Hey Gus, I'm getting complains that the site is loading way too slowly. I
thought it was happening just to me because I;m doing editing but
apparently it's taking more than a minute to load a single page for other
people. We need to fix this ASAP.

**Comment by gusaus** [comment]
Hey Rocco - I went back and compressed all the images on the event pages
(including featured images) - that boosted performance a bit but I'll also
try to spend some time this afternoon seeing if I can further improve.

**Comment by gusaus** [comment]
Hey Rocco - I was doing a quick check on the site and noticed the YouTube vids on this page are no longer available - 
https://angelcityjazz.com/explorations/la-creators-eric-dolphy/

Let me know if you'd like me to remove them until you're able to swap out with new ones.

**Comment by angelcityjazz** [comment]
I’ll look for alternative videos. You can leave it as is for now.
Thanks for catching that.

**Comment by gusaus** [comment]
@angelcityjazz Reopening this so we can evaluate and sign off on https://acjdev.wpengine.com/explorations/

Main thing right now is to make sure we're happy with the general layout and formatting (spacing, colors, buttons, etc.).

I'll also add the latest Spotify embed but I'm realizing that will be an odd number. Wondering if there's now enough content to split out into multiple sub-pages (something like 'Read' and 'Listen') that we could then have as submenu items for 'Explorations'. Similar to what we're doing with https://acjdev.wpengine.com/watch/

Again, probably not something I could do before going live, but soon-after if you think that might work.

Do you see any glaring problems or quick changes you'd like to see? Leave a comment and I'll review and work in before pushing to live tonight.

**Comment by angelcityjazz** [comment]
I’m open to splitting  the playlists to a separate page.
